src/styles/index.scss

```diff
@@ -151,9 +155,29 @@ td {
 
 }
 
+.button-message {
```

Comment:
These widths are keeping the table 'static' so they no longer move and cause a user annoyance when they add or subtract ingredients.